pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools >= 41", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 79
+skip-string-normalization = true
+target-version = ['py36', 'py37', 'py38', 'py39']
+exclude = '''
+/(
+    \.git
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | \.pytest_cache
+  | dist
+  | build
+  | docs
+)/
+'''
+
+# This next section only exists for people that have their editors
+# automatically call isort, black already sorts entries on its own when run.
+[tool.isort]
```

Comment:
This should just be `profile = black` and drop everything but the `line_length`.